python/pyspark/ml/classification.py

```diff
@@ -320,8 +322,13 @@ class RandomForestClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPred
     >>> allclose(model.treeWeights, [1.0, 1.0, 1.0])
     True
     >>> test0 = sqlContext.createDataFrame([(Vectors.dense(-1.0),)], ["features"])
-    >>> model.transform(test0).head().prediction
+    >>> result = model.transform(test0).head()
+    >>> result.prediction
     0.0
+    >>> numpy.argmax(result.probability)
+    0
+    >>> numpy.argmax(result.rawPrediction)
+    0
```

Comment:
Here users may want to know the raw prediction and probability prediction result as the confidence measure, so I think output ```result.probability``` or ```numpy.amax(result.probability)``` is better to demonstrate how to use this feature.